### PR TITLE
Updating and reorganizing the setup instructions for the monorepo and wallet

### PR DESCRIPTION
**Pull request by tarikbellamine**
## Description

Updated and reorganized the onboarding documents found on the monorepo SETUP.md and the mobile directory README.md. Goal is to separate monorepo setup and mobile wallet setup and to improve the layout and clarify of the onboarding instructions. 

## Other changes

N/A

## Tested

Clicked through most the links and proofread the changes. Did not attempt to re-setup in a clean environment because I did not add/remove any specific installs.

## Related issues

No open ones I know of.

## Backwards compatibility

N/A

## Commit Message
Updating monorepo SETUP.md and mobile package README.md

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -2,195 +2,107 @@
...
 nvm install 10
 nvm alias default 10
-
-# Installing Yarn - https://yarnpkg.com/en/docs/install#debian-stable
-curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-sudo apt-get update && sudo apt-get install yarn
-
-# Install typescript
 npm install -g typescript
```
Comment (jmrossy) [inline SETUP.md]: Pretty sure we don't require a global typescript installation, not sure why this line is here

File: SETUP.md [hunk 2]
```diff
@@ -2,195 +2,107 @@
...
-# Installing Node
+# Installing NodeJS
 sudo apt-get update
 sudo apt-get install nodejs
+```
 
-# Installing Nvm
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
-source ~/.bashrc
+Install `nvm` (allows you to manage multiple versions of Node) by following the instructions here: [https://github.com/nvm-sh/nvm].
```
Comment (jmrossy) [inline SETUP.md]: How about we DRY this part up and move it common stuff?

File: SETUP.md [hunk 3]
```diff
@@ -314,45 +179,14 @@ yarn build # build all packages
...
 > and the `postinstall` step which uses it is not automatically run after using `yarn remove`.
 
-## Using an Android test device locally
-
-First, follow [these instructions to enable Developer Options][android dev options]
-on your Android.
 
-Plug in a USB cable and you'll be prompted to accept the connection and shown a
-public key (corresponding to the `abd_key.pub` file in `~/.android`)
+## Deploying the mobile wallet
```
Comment (jmrossy) [inline SETUP.md]: Consider renaming to 'Running the mobile wallet'

File: SETUP.md [hunk 4]
```diff
@@ -314,45 +179,14 @@ yarn build # build all packages
...
 
-Then, running:
+To deploy the mobile wallet to your iOS or Android devices see the guide in the [mobile](packages/mobile) directory.
```
Comment (jmrossy) [inline SETUP.md]: Ditto, 'To build and run the mobile wallet...'

**Review by jmrossy — COMMENTED** [review]
Looks great! Just a few small comments

File: SETUP.md (hunk 1 above)
Comment (tarikbellamine) [reply SETUP.md]: I'm not sure either but am missing the historical context. I'm good with removing it if don't think we need it.

File: SETUP.md (hunk 2 above)
Comment (tarikbellamine) [reply SETUP.md]: Good idea. Will also move "common stuff" to before MacOS/Linux specific instructions so the ordering doesn't get messed up.

File: SETUP.md (hunk 1 above)
Comment (jmrossy) [reply SETUP.md]: Yeah lets remove it

File: SETUP.md (hunk 2 above)
Comment (jmrossy) [reply SETUP.md]: Cool but maybe then break out the optional parts?

**Review by jmrossy — APPROVED** [review]
Ship it 🚢

**Pull request merged**